app/Http/Kernel.php

```diff
@@ -25,6 +25,11 @@ class Kernel extends HttpKernel
      * @var array<string, array<string|class-string>>
      */
     protected $middlewareGroups = [
+        'web' => [
+            \Illuminate\Cookie\Middleware\EncryptCookies::class,
+            \Illuminate\Session\Middleware\StartSession::class,
+            \Illuminate\Foundation\Http\Middleware\VerifyCsrfToken::class,
+        ],
```

Comment:
This is what Laravel project scaffolding (`$ laravel new myapp`) gave me:
```php
    'web' => [
            \App\Http\Middleware\EncryptCookies::class,
            \Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse::class,
            \Illuminate\Session\Middleware\StartSession::class,
            // \Illuminate\Session\Middleware\AuthenticateSession::class,
            \Illuminate\View\Middleware\ShareErrorsFromSession::class,
            \App\Http\Middleware\VerifyCsrfToken::class,
            \Illuminate\Routing\Middleware\SubstituteBindings::class,
        ],
```

Not sure if it matters. Sanctum's `Laravel\Sanctum\Http\Middleware\EnsureFrontendRequestsAreStateful` middleware in `config/lighthouse.php` [includes some of that middleware](https://github.com/laravel/sanctum/blob/2.x/src/Http/Middleware/EnsureFrontendRequestsAreStateful.php#L28-L31), most importantly `AddQueuedCookiesToResponse` and `StartSession`.

Reply:
Also, a couple lines below should be `'guard' => 'sanctum'` (I could not annotate the line directly because it wasn't changed):

https://github.com/nuwave/lighthouse-example/blob/2e7ade3bec574290fdbce8f498d663bfaea6fa89/config/lighthouse.php#L48-L59

Reply:
I guess `AddQueuedCookiesToResponse` should be added, then. Or how about we just add `Laravel\Sanctum\Http\Middleware\EnsureFrontendRequestsAreStateful` itself?

Reply:
The sanctum middleware should only be added to `config/lighthouse.php` for the scope of this example. Otherwise, all other routes would be protected as well. I guess you can just use the middleware that scaffolding gives you.